src/openrct2/actions/RideEntranceExitRemoveAction.hpp

```diff
@@ -110,7 +107,12 @@ DEFINE_GAME_ACTION(RideEntranceExitRemoveAction, GAME_COMMAND_REMOVE_RIDE_ENTRAN
             break;
         } while (!(tileElement++)->IsLastForTile());
 
-        if (!found)
+        // Silently prevent removing an existing entrance/exit if we're trying to place a ghost
```

Comment:
This is the remove action this doesn't make much sense. We aren't trying to place a ghost we are trying to remove a ghost but we have a real tile element here instead. Can you change the comment and use a different GA_ERROR as I don't see it as the correct one to use for this.

Reply:
Reworded so hopefully that's a bit clearer, and changed the error to `GA_ERROR::INVALID_PARAMETERS`.